specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/examples/SuccessfulAspectSentimentRequest.json

```diff
@@ -0,0 +1,315 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "Endpoint": "{Endpoint}",
+    "input": {
+      "documents": [
+        {
+          "language": "en",
+          "id": "1",
+          "text": "Great atmosphere. Close to plenty of restaurants, hotels, and transit! Staff are friendly and helpful."
+        },
+        {
+          "language": "en",
+          "id": "2",
+          "text": "Bad atmosphere. Not close to plenty of restaurants, hotels, and transit! Staff are not friendly and helpful."
+        }
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "documents": [
+          {
+            "id": "1",
+            "sentiment": "positive",
+            "documentScores": {
+              "positive": 0.998519241809845,
```

Comment:
Aren't we supposed to show score up to 2 decimal places?

Reply:
I have not changed to the sentiment results yet; only aspect related results are correct so far.